srdt_analysis/llm_runner.py

```diff
@@ -67,9 +68,14 @@ async def rephrase_and_split(
     async def chat_with_full_document(
         self,
         chat_history: list[UserLLMMessage],
-        prompt: str,
+        optional_system_prompt: Optional[str] = None,
```

Comment:
```suggestion
        system_prompt: Optional[str] = None,
```

Pas besoin de mettre `optional` dans le nom de paramètre je pense, le type le montre déjà d'ailleurs

Reply:
D'acc, je vais appliquer cette logique sur d'autres types alors

Reply:
Done here : https://github.com/SocialGouv/srdt/pull/75/commits/ed82b26f282dea910129a5c0fcca4f313ba90b27